src/__swaps__/screens/Swap/components/TokenList/TokenToBuyList.tsx

```diff
@@ -1,34 +1,221 @@
-import { StyleSheet } from 'react-native';
-import React from 'react';
-import { TokenToBuySection } from '@/__swaps__/screens/Swap/components/TokenList/TokenToBuySection';
-import { Stack } from '@/design-system';
+import { FlashList } from '@shopify/flash-list';
+import React, { memo, useCallback, useMemo } from 'react';
+import Animated, { runOnUI, useAnimatedStyle, withTiming } from 'react-native-reanimated';
+import * as i18n from '@/languages';
+import { AnimatedTextIcon } from '@/components/AnimatedComponents/AnimatedTextIcon';
+import { TIMING_CONFIGS } from '@/components/animations/animationConfigs';
+import { Box, Inline, Stack, Text, TextIcon, useColorMode } from '@/design-system';
+import { palettes } from '@/design-system/color/palettes';
+import { userAssetsStore } from '@/state/assets/userAssets';
+import { COIN_ROW_WITH_PADDING_HEIGHT, CoinRow } from '@/__swaps__/screens/Swap/components/CoinRow';
 import { ListEmpty } from '@/__swaps__/screens/Swap/components/TokenList/ListEmpty';
+import { AssetToBuySectionId, useSearchCurrencyLists } from '@/__swaps__/screens/Swap/hooks/useSearchCurrencyLists';
+import { useSwapContext } from '@/__swaps__/screens/Swap/providers/swap-provider';
+import { ChainId } from '@/__swaps__/types/chains';
+import { SearchAsset } from '@/__swaps__/types/search';
+import { SwapAssetType } from '@/__swaps__/types/swap';
+import { parseSearchAsset } from '@/__swaps__/utils/assets';
+import { getStandardizedUniqueIdWorklet } from '@/__swaps__/utils/swaps';
+import { DEVICE_WIDTH } from '@/utils/deviceUtils';
+import { EXPANDED_INPUT_HEIGHT, FOCUSED_INPUT_HEIGHT } from '../../constants';
 import { ChainSelection } from './ChainSelection';
-import { useSearchCurrencyLists } from '../../hooks/useSearchCurrencyLists';
+
+const BUY_LIST_HEADER_HEIGHT = 20 + 10 + 8; // paddingTop + height + paddingBottom
```

Comment:
move to constants.ts

Reply:
Same thoughts here, leaving since it's a local const and specific to the layout of this particular list (bottom padding is different between buy/sell lists).